rest/service.go

```diff
@@ -233,6 +233,7 @@ func (s *Service) addRoutes() {
 	s.app.AddRoute("/perf/version/{version}").Version(1).Get().RouteHandler(makeGetPerfByVersion(s.sc))
 	s.app.AddRoute("/perf/children/{id}").Version(1).Get().RouteHandler(makeGetPerfChildren(s.sc))
 	s.app.AddRoute("/perf/change_points").Version(1).Post().Wrap(checkUser).RouteHandler(makePerfSignalProcessingRecalculate(s.sc))
+	s.app.AddRoute("/perf/project/{projectId}/version/change_points").Version(1).Get().RouteHandler(makeGetChangePointsByVersion(s.sc))
```

Comment:
version seems weird in this path, but I'm not sure what I'd put?

Reply:
Yeah, I'm not sure of a better way to describe it. It's currently listed in the design as this so that's what I went with

Reply:
can we change it to /perf/project/{projectID}/change_points_by_version

Reply:
@ancostas any thoughts?

Reply:
For a single version we'd be doing `/perf/project/{p_id}/version/{v_id}/change_points`, right?

Wouldn't the bulk route then ideally be `/perf/project/{projectId}/version/change_points` for multiple versions w/ change points under them?

Reply:
Separately, FWIW, I think it would be nice if we could just go with the design doc's prescription. There exists a case where the UI consuming this route had already been developed, and we'd basically be breaking it and requiring an update in a separate codebase. That's not happenening in this case, but something I think is worth keeping in mind -- this is why I try to include exact prescriptions for the APIs being developed in my designs, without saying much about how exactly to implement them.

Reply:
we haven't used that paradigm for aggregate routes anywhere in cedar or evergreen. 

what about /perf/project/{projectID}/version/{versionID}/change_points and /perf/project/{projectID}/change_points

Reply:
I mean, to me the route is "Get versions with change points under a specific project", so it seems like the resource would be /projects/p_id/versions/change_points. Or /projects/p_id/versions?include=change_points?

I don't really care though, other than I find diverging from the agreed design a bit hard to scale if we had more people working on this asynchronously. Since we don't, go ahead and make the route whatever makes sense to you two, but please update the technical design such that whomever catches the UI ticket for this  knows what the correct API endpoint is.

Reply:
I think Alex's take on how to think about this route is the correct one. This does seem to be very contentious though and I think is one of the last pieces to resolve on this PR. If you want to discuss further Sam, we can set up a quick call to talk through and hopefully come to an agreement.